packages/SwingSet/docs/delivery.md

```diff
@@ -589,19 +596,20 @@ resolves, or both. To reduce the noise of unwanted notifications, Vats will
 not receive a `dispatch.notify()` for a Promise unless they first use
 `syscall.subscribe()` to express their interest.
 
-The `PromiseID` argument to `subscribe()` is translated through the c-list
+The `PromiseID` argument to `subscribe()` is translated through the C-List
 just like a `CapSlot` in `syscall.send()`. It is not common for this to cause
 the allocation of a `KernelPromise`, because Vats don't usually subscribe to
-hear about their own Promises, but it is legal.
+hear about their own Promises, but it is valid.
 
 ### syscall.resolve()
 
-`syscall.resolve` is used to resolve one or more Promises (usually just one,
+`syscall.resolve()` is used to resolve one or more Promises (usually just one,
 but occasionally a batch of mutually-referencing Promises must be resolved in
 a single syscall because their identifiers are aggressively retired
-immediately after translation).
+immediately after translation). \[TODO: resolve inconsistency between this claim
+and [Syscall/Dispatch API](#syscalldispatch-api)]
```

Comment:
The inconsistency is documentation of `trait Syscall` including `fn resolve(subject: PromiseID, to: Resolution)`, which seems unable to support resolving multiple promises in a batch.